contents/blog/recommended-reading.md

```diff
@@ -0,0 +1,148 @@
+**Recommended by [James Hawkins, CEO & Co-founder](/community/profiles/71)**
+
+James reads a lot and could likely recommend enough books to fill an article all on his own — but when we asked for recommendations for first-time founders, this was one of the first books which came to mind. It focuses on explaining how startups operate from a VC’s perspective.
+
+“Sand Hill Road was helpful for me when we started PostHog because we didn’t have any Silicon Valley insiders in our network,” he explained. “It taught me that all the returns in venture are based on just a tiny handful of companies — they simply can’t make their model work from companies that don’t totally smash it.” 
+
+“If you’re thinking about how to finance a business, it helps to be naturally aligned with this — which we are, it turns out!”
+
+> **Putting it into practice:** Understanding how VCs think about startup has helped us raise funding, even when [we’ve decided to do it ahead of schedule](/blog/why-we-raised-a-15m-series-b-ahead-of-schedule)!
+
+![messy middle](../images/tracks/messy-middle.jpg)
+
+## [The Messy Middle by Scott Belsky](https://www.goodreads.com/book/show/40179007-the-messy-middle)
+**Recommended by [Charles Cook, Operations & Marketing](/community/profiles/254)**
+
+If there’s one person whose book recommendations are really worth listening to, then it’s Charles. Not only is he the leader of both our Operations and Marketing teams — he’s _also_ the founder of our internal book club.
+
+“I like The Messy Middle because, as the title suggests, it covers the really important interim period for startups that are starting to scale,” says Charles. “Most startup type books cover either the very beginning when you’re validating your idea, or the later stages when you are 50+ people. This book is full of specific, actionable advice for the time in between those stages.”
+
+> **Putting it into practice:** When it comes to ops, there’s no silver bullet. Instead, [you need a whole toolkit](/blog/startup-ops-toolkit) to help you solve messy problems!
+
+
+![high output management](../images/tracks/high-output.jpg)
+## [High Output Management by Andrew Grove](https://www.goodreads.com/book/show/324750.High_Output_Management)
+**Recommended by [Grace McKenzie, Finance & Operations Lead](/community/profiles/718)**
+
+Andrew Grove was Intel’s third employee, third CEO, and the man who led the company to the world’s largest producer of semiconductors. He also somehow found time to write this book about organizational theory which is beloved by our chief organizer, Grace.
+
+“High Output Management is filled with light-touch and thoughtful management advice which can lead to increases in productivity,” says Grace. “In particular it advocates for the importance of gathering relevant information, making decisive decisions, and nudging teams towards a goal rather than setting a specific direction.”
```

Comment:
```suggestion
**Grace's take:** “High Output Management is filled with light-touch and thoughtful management advice which can lead to increases in productivity. In particular it advocates for the importance of gathering relevant information, making decisive decisions, and nudging teams towards a goal rather than setting a specific direction.”
```